src/styles/blocks/star.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+  }
+
+  &--1 &__star:nth-child(-n + 1) {
+    background-image: $star-img;
+  }
+
+  &--2 &__star:nth-child(-n + 2) {
+    background-image: $star-img;
+  }
```

Comment:
It's recommended to be consistent with the use of `nth-child` and `nth-of-type`. Here, `nth-child` is used, but in lines 21, 25, and 29, `nth-of-type` is used. Choose one method and use it consistently throughout the code.